app/controllers/api/v1/users_controller.rb

```diff
@@ -1,9 +1,8 @@
 module Api
   module V1
     class UsersController < V1::BaseController
-      include Foreman::Controller::UsersMixin
-
       before_filter :find_resource, :only => %w{show update destroy}
+      include Foreman::Controller::UsersMixin
```

Comment:
please add it, as the failing test in this case won't make the cause clear I think